draft-scenarios/campaigns/castle_two_knights_defense.tex

```diff
@@ -0,0 +1,243 @@
+\end{itemize}
+
+\begin{tikzpicture}[overlay]
+    \node at (-1.0, -9.0) {
+      \includegraphics[width=0.70\paperwidth]{\_assets/maps/castle_two_knights_defense.png}
+    };
+  \end{tikzpicture}
+\end{multicols}
+
+\newpage
+
+\begin{multicols*}{2}
+
+\subsection*{\MakeUppercase{The story}}
+
+\textbf{The Race}
+
+Stacks of dry, cracked parchment sat next to dusty tomes on a table's blackened, rough-cut stone slab.
+Catherine slumped over the numerous texts stashed throughout the necromancer's chambers while her eyes frantically scanned the pages.
+She was searching for any clue to her enemy's plans.
+
+After fruitless hours, she huffed and sent a moldering volume crashing through a stack of paper as exhaustion threatened to overcome her.
+She was about to spin and leave the room when the corner of her eye caught a particular scribbling, revealed by her destructive act.
+An ornate diagram, hastily scribbled, depicted what Catherine supposed might be some kind of necromantic summoning ritual.
+Pulling the frail parchment closer, while most of it didn't make sense to her, it didn't take a mage to know what the dark drawings detailed.
+They were the ritual instructions for raising Ghost Dragons from dead dragons.
+
+Then it dawned on her.
+Old tales told of an ancient Dragon Utopia in the badlands.
+Vhidhax must be attempting to overtake the majestic creatures and use them against her in a last-ditch effort.
+She frowned at the thought of her weakened enemy being able to defeat the dragons.
+But if he did… If she missed something, he would have newfound strength enough to overpower her battered forces.
+She clenched her fist, crushing the frail paper to dust before rushing to the doorway to gather who she could.
+
+To her surprise, the necropolis' courtyard was a buzz of energy.
+She placed her hands on the overlook's railing and peered over the edge to spy a host of knights in the color and flair of various baronies. 
+
+Rion came rushing up to her a moment later, intent on delivering news of their new guests.
+``Queen Catherine!'' he said excitedly.
+``Lordships around Steadwick heard of the tragedy and have sent their champions to aid us.
+The knights spent days tracking and catching up to us, and they mean to join us and see the campaign through.''
+
+Catherine smiled wide.
+``Then tell them to get ready; the race is on.''
+
+\textcolor{darkcandyapplered}{Add ``A Few'' Champions to your army.}
+
+\textbf{Checkmate}
+
+The broken remains of Vhidhax looked no grander than any of his many minions.
+Necromantic energy stripped the flesh from him long ago; without that power, his presence and ill-willed malevolence were gone as well.
+
+Catherine wiped blood, sweat, and dirt from her face.
+Rion had been right; the necromancer was not her father, but perhaps his slaying would ease the sting of her family discourse and the sting of betrayal, at least a little.
```

Comment:
```suggestion
Rion had been right; the necromancer was not her father, but perhaps his slaying would ease the sting of her family discord and the sting of betrayal, at least a little.
```
There's a fine line between those two words, don't you think?

Reply:
Absolutely, maybe I was thinking of the wrong one